### PR TITLE
Specified date feature as requested

### PR DESCRIPTION
**Pull request by KanadeFuzen**
## Description

Simple small feature name change that was requested from the Trello.

## Fixes

Was requested that we should specify either “hearing date” or “decision date.” Since, we are using decision date and it is easier to scrape for, obviously it was changed to decision date.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: SCRAPER_INFO.md [hunk 1]
```diff
@@ -11,7 +11,7 @@
 - **Current Status:** Currently is pulled correctly by scraper.
 
 ### Date
-- **Function name: ** `get_date()`
+- **Function name: ** `decision_date()`
```
Comment (minh14496) [inline SCRAPER_INFO.md]: The function still name get_date() because u haven't change the name in ocr.py. You only change the column name of the table

**Review by minh14496 — CHANGES_REQUESTED** [review]

File: SCRAPER_INFO.md (hunk 1 above)
Comment (KanadeFuzen) [reply SCRAPER_INFO.md]: Oh my bad, I might have accidently wrote that over when using ctrl-f in the md file. I should just revert that line back to get_date() right?

Comment (minh14496) [reply SCRAPER_INFO.md]: Yes Waqas

**Review by minh14496 — APPROVED** [review]
Let's go Waqas. We need that credibility function

**Review by andrewlee977 — APPROVED** [review]
Looks good, Waqas!

**Pull request merged**